### PR TITLE
A step added to build/run examples

### PR DESCRIPTION
**Pull request by oleksandr-pavlyk**
`conda-packages.yaml` workflow add a step to build and run dpctl/examples on Linux.

This PR is to incorporate exercising examples to make sure that they are not broken by future changes.

### CODE REVIEW

File: .github/workflows/conda-package.yml [hunk 1]
```diff
@@ -112,10 +116,11 @@ jobs:
         uses: actions/download-artifact@v2
         with:
           name: ${{ env.PACKAGE_NAME }} ${{ runner.os }} Python ${{ matrix.python }}
-      - name: Add conda to system path
-        run: echo $CONDA/bin >> $GITHUB_PATH
       - name: Install conda-build
+        # Needed to be able to run conda index
         run: conda install conda-build
+      - name: Add conda to system path
+        run: echo $CONDA/bin >> $GITHUB_PATH
```
Comment (PokhodenkoSA) [inline .github/workflows/conda-package.yml:123]: Is reordering of this steps necessary?

Comment (oleksandr-pavlyk) [reply .github/workflows/conda-package.yml:123]: Not at all.

**Review by PokhodenkoSA — DISMISSED** [review]
LGTM if it works. 
I think overall conda-package workflow looks messy and needs refactoring. But it is separate story.

File: .github/workflows/conda-package.yml (hunk 1 above)
Comment (oleksandr-pavlyk) [reply .github/workflows/conda-package.yml]: Order restored.

**Review by PokhodenkoSA — APPROVED** [review]

**Review by diptorupd — APPROVED** [review]

**Pull request merged**